www/docs/en/dev/config_ref/index.md

```diff
@@ -45,207 +45,356 @@ passively copied into various `platforms/` subdirectories, for example:
         app/platforms/blackberry10/www/config.xml
         app/platforms/android/res/xml/config.xml
 
-This section details global and cross-platform configuration options.
-See the following sections for platform-specific options:
-
-- [iOS Configuration](../guide/platforms/ios/config.html)
-- [Android Configuration](../guide/platforms/android/config.html)
-- [BlackBerry 10 Configuration](../guide/platforms/blackberry10/config.html)
-
 In addition to the various configuration options detailed below, you
 can also configure an application's core set of images for each target
 platform. See [Icons and Splash Screens](images.html) for more information.
 
-## Core Configuration Elements
-
-This example shows the default `config.xml` generated by the CLI's
-`create` command, described in [The Command-Line Interface](../guide/cli/index.html):
-
-        <widget id="com.example.hello" version="0.0.1">
-            <name>HelloWorld</name>
-            <description>
-                A sample Apache Cordova application that responds to the deviceready event.
-            </description>
-            <author email="dev@callback.apache.org" href="http://cordova.io">
-                Apache Cordova Team
-            </author>
-            <content src="index.html" />
-            <access origin="*" />
-        </widget>
-
-The following configuration elements appear in the top-level
-`config.xml` file, and are supported across all supported Cordova
-platforms:
-
-- The `<widget>` element's `id` attribute provides the app's
-  reverse-domain identifier, and the `version` its full version number
-  expressed in major/minor/patch notation.
-
-  The widget tag can also have attributes that specify alternative versions,
-  namely `versionCode` for Android, `CFBundleVersion` for iOS, and 
-  `packageVersion` for Windows. See the Additional Versioning section below 
-  for details.
-
-- The `<name>` element specifies the app's formal name, as it appears
-  on the device's home screen and within app-store interfaces.
-
-- The `<description>` and `<author>` elements specify metadata and
-  contact information that may appear within app-store listings.
-
-- The optional `<content>` element defines the app's starting
-  page in the top-level web assets directory. The default value is
-  `index.html`, which customarily appears in a project's top-level
-  `www` directory.
-
-- `<access>` elements define the set of external domains the app is
-  allowed to communicate with. The default value shown above allows
-  it to access any server. See the Domain [Whitelist Guide](../guide/appdev/whitelist/index.html) for details.
-
-- The `<preference>` tag sets various options as pairs of
-  `name`/`value` attributes. Each preference's `name` is
-  case-insensitive.  Many preferences are unique to specific
-  platforms, as listed at the top of this page. The following sections
-  detail preferences that apply to more than one platform.
-
-### Additional Versioning
+# widget
```

Comment:
I followed the format of <tag-name> as heading i.e. <widget> here, for plugin.xml. What do you think is better? We should be consistent.